org-roam-bibtex-note-actions.el

```diff
@@ -199,15 +199,9 @@ that can be customized.  Additionally, user actions can be set in
 `org-roam-bibtex-note-actions-user'."
   (interactive)
   (let ((non-default-frontends (list 'hydra 'ido 'ivy 'helm))
-        (citekey (cdr (assoc "ROAM_KEY" (org-roam--extract-global-props '("ROAM_KEY"))))))
-    ;; remove format from citekey
-    (when org-roam-bibtex-citekey-format
-      (string-match "\\(.*\\)%s\\(.*\\)" org-roam-bibtex-citekey-format)
-      (let ((beg (match-end 1))
-            (end (+ (length citekey)
-                    (- (match-beginning 2)
-                       (length org-roam-bibtex-citekey-format)))))
-        (setq citekey (substring citekey beg end))))
+        (citekey (cdr (assoc "ROAM_KEY"
+                             (org-roam--extract-global-props
+                              '("ROAM_KEY"))))))
```

Comment:
I think the merge erased the work you'd done on c52e3bf6475aa94c43123b1f02b7fd4b77ac6270.

Reply:
Redid the merge in 7754381a937a9a858d0ac564a2340fa1041b6d6d.

I had to rebase, so make sure to pull before you start any further work.